client/components/favorites/Favorites.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Recipe from '../recipe/Recipe';
+
+const propTypes = {
+  favorites: PropTypes.arrayOf(PropTypes.object).isRequired,
+};
+
+class Favorites extends Component {
```

Comment:
This should be rewritten to be a stateless functional component seeing as it maintains no state of it's own nor does it use any of the lifecycle methods